metadata-ingestion/src/datahub/ingestion/source/dbt.py

```diff
@@ -104,13 +116,13 @@ def get_columns(catalog_node: dict, manifest_node: dict) -> List[DBTColumn]:
 
     for key in raw_columns:
         raw_column = raw_columns[key]
-
         dbtCol = DBTColumn(
             name=raw_column["name"].lower(),
             comment=raw_column.get("comment", ""),
             description=manifest_columns.get(key.lower(), {}).get("description", ""),
             data_type=raw_column["type"],
             index=raw_column["index"],
+            tags=manifest_columns.get(key.lower(), {}).get("tags", []),
```

Comment:
since these are tags being imported from dbt, could you prefix them with "dbt:" so that they don't collide with tags imported from other places / being authored directly in datahub?
maybe create a config variable that controls this feature: `imported_tags_prefix` that defaults to `dbt:` but can be set to "" to achieve the behavior you are showing here.  

This would also mean sending the tag mce-s for all the tags that you have discovered during the ingestion.

Reply:
I can implement the second option, since what we want is to have the same tags across "platforms" to be able to search on tags. Hope that is ok :) 

I'm currently also setting the globalTags, should I change this then? The tags gets inserted to datahub and are searchable and such. What will change if I actually send the mce-s?

Reply:
Implementing second option is fine!

I didn't see you creating the Tag entity itself in this PR. Are you saying that is not needed?

Reply:
I ran everything locally and did an ingest and all the tags got created and I could search by them. Not sure what else the tag creation itself would do but maybe it’s something I’m missing :)

Reply:
@shirshanka So, should I create the tags themselves or not? :)

Reply:
I think the GlobalTagsClass I create is enough. But maybe someone else could share some light on this?